requirements.txt

```diff
@@ -1,3 +1,6 @@
 jsonschema==3.2.0
 boto3==1.18.22
 backoff==1.11.1
+pytest-cov
+moto[s3]
```

Comment:
Those are already in the `[options.extras_require]` section L34-35. I keep them apart as they are not needed to run the package, only to run the tests.
My rational is to keep the target lean if possible.

So I don't know how much of a best practice it is but I used
```python
pip install --upgrade .[test,lint,dist]
```
To install what's needed on `local`. It was my way of following the KISS principle here.